frontend/src/components/TeledeclarationPreview.vue

```diff
@@ -17,6 +17,16 @@
               </tr>
             </thead>
             <tbody>
+              <tr>
+                <td class="text-left font-weight-bold">
+                  Données relatives à votre établissement
+                </td>
+                <td class="text-left font-weight-bold"></td>
+              </tr>
+              <tr v-for="item in canteenItems" :key="item.label">
+                <td class="text-left">{{ item.label }}</td>
+                <td :class="diagnostic[item.param] ? 'text-right' : 'text-left'">{{ item.value }}</td>
```

Comment:
item.param doesn't appear to be used but it would be cool to have it for the items that are numbers like meal and satellite counts

Reply:
Good catch : https://github.com/betagouv/ma-cantine/pull/2301/commits/3987c9b13a8151b46372ef01df3c6bfc29d0ad0f